contrib_bots/bots/thesaurus/thesaurus.py

```diff
@@ -21,24 +21,26 @@ def get_clean_response(m, method):
         return response
     elif isinstance(response, list):
         return ', '.join(response)
+    else:
+        return "Not found"
```

Comment:
Nit: Can you please keep the original nice message "Sorry, no result found! Please check the word."?